tests/kernel/fatal/protection_no_userspace.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_ZTEST=y
+CONFIG_COVERAGE=n
```

Comment:
why is COVERAGE explicitly disabled?

Reply:
no particular reason @andrewboie , I suppose it is the same in the default prj.conf